tests/codegen/riscv-target-abi.rs

```diff
@@ -0,0 +1,20 @@
+//@ revisions:riscv64gc riscv32gc riscv32imac
+
+//@[riscv64gc] compile-flags: --target=riscv64gc-unknown-linux-gnu
+//@[riscv64gc] needs-llvm-components: riscv
+// riscv64gc: !{i32 {{1}}, !"target-abi", !"lp64d"}
```

Comment:
```suggestion
// riscv64gc: !{i32 1, !"target-abi", !"lp64d"}
```
Why the `{{1}}`?

Reply:
Fixed. I directly copied it from the output of `llvm-dis`.